client/payment-methods-map.tsx

```diff
@@ -168,6 +169,20 @@ const PaymentMethodInformationObject: Record<
 		allows_pay_later: false,
 		accepts_only_domestic_payment: false,
 	},
+	multibanco: {
+		id: 'multibanco',
+		label: __( 'Multibanco', 'woocommerce-payments' ),
+		description: __(
+			'Allow customers to pay with Multibanco, a popular payment method in Portugal.', // TODO: Get the correct description.
```

Comment:
Do you happen to have a follow-up for this?

FWIW, I've been using a slightly modified description from the Stripe plugin :) 

https://github.com/woocommerce/woocommerce-gateway-stripe/blob/develop/includes/payment-methods/class-wc-stripe-upe-payment-method-multibanco.php#L27